docs/userguide.rst

```diff
@@ -241,6 +241,37 @@ analysis inputs, and start the workflow run.
    myanalysis   1            2018-11-07T12:45:18   running   1/1
    $ reana-client download results/plot.png
 
+Running specific parts of analysis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Serial*
+
+Serial workflows can be executed partially until the step specified by the user.
+To do so, you need to provide target step name as operational option for the
+``reana-client start`` or ``reana-client run`` commands.
+
+.. code-block:: console
+
+   $ reana-client start -w workflow.1 -o target='gendata'
+   # or
+   $ reana-client run -w workflow.1 -o target='gendata'
+
+*CWL*
+
+CWL allows to execute only certain step of the workflow specified by the user.
+To do so, you need to provide target step name as operational option for the
+``reana-client start`` or ``reana-client run`` commands.
```

Comment:
What do you think of pointing to CWL docs here? Strictly speaking, the `target='gendata'` means something different for serial than for CWL, in serial runs until `genedata` for CWL it runs from `gendata`.

> If a target is an output parameter, it will only run only the steps that contribute to that output. If a target is a workflow step, it will run the workflow starting from that step. If a target is an input parameter, it will only run only the steps that are connected to that input.

Extracted from https://github.com/common-workflow-language/cwltool#running-only-part-of-a-workflow.